practice-two/src/index.html

```diff
@@ -1,187 +1,336 @@
-            </div>
+            <a href="javascript:void(0)" class="testimonials__link">
+              <svg class="testimonials__icon">
+                <use
+                  xlink:href="assets/images/icon-testimonial.svg#icon-next"
+                ></use>
+              </svg>
+            </a>
           </div>
         </div>
-      </div>
-    </section>
-
-    <!-- END ADVANTAGES SECTION-->
+      </section>
+      <!-- END TESTIMONIALS SECTION-->
 
-    <!-- START TESTIMONIALS SECTION -->
-    <section class="testimonials">
-      <div class="container testimonials__wrapper">
-        <h2 class="heading-section text-color-base testimonials__header-text">What Clients are saying</h2>
-        <div class="testimonials__content">
-          <a href="javascript:void(0)" class="testimonials__link">
-            <svg class="testimonials__icon">
-              <use xlink:href="assets/images/icon-testimonial.svg#icon-prev"></use>
-            </svg>
-          </a>
-          <div class="testimonials__item">
-            <div class="testimonials__description">
-              <p class="text-color-base testimonials__description-text">Businesses at present had creatively generated effective
-                advertising campaigns in the form of audio visual media that includes the television and radios and the print media that cover the newspaper and
-                campaign materials like the posters, brochures, business cards, flyers and many more.
-              </p>
+      <!-- START PROJECTS SECTION -->
+      <section class="text-color-secondary project">
+        <div class="container">
+          <hgroup class="project__header">
+            <h2 class="heading-section">
+              Recent Products
+            </h2>
+            <p class="text-normal project__header-text">
+              Having used discount toner cartridges for twenty years, there have
+              been a lot of changes in the toner cartridge market. The market
+              today is approximately a twenty billion dollar business.
+            </p>
+          </hgroup>
+          <div class="project__contents">
+            <div class="project__item">
+              <figure class="project__item-img">
+                <img
+                  src="assets/images/image-product-1.webp"
+                  alt="thump FTA key image"
+                  class="img-default img-border"
+                />
+                <div class="project__view">
+                  <p class="text-normal text-color-base project__view-counts">91312</p>
+                </div>
+              </figure>
+              <hgroup class="project__item-description">
+                <h3 class="project__item-title">App maintenance</h3>
+                <p class="text-normal project__item-author">
+                  <span class="project__item-label">by</span> Lida Morrison, Nettie Burke
+                </p>
+              </hgroup>
+              <button class="btn btn-base project__btn">review</button>
```

Comment:
You use project__btn to change background color for button, that means you change behavior for the button. You should use modifier for class here. It should be `btn---project`, so on

Same for btn-base and btn-primary, btn-secondary. It should be btn--base, btn--primary, btn--secondary. 

You can refer to the example at https://sparkbox.com/foundry/bem_by_example